Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -397,7 +397,44 @@ public static void HandleSmite(Session session, params string[] parameters)
         {
             // @smite [all] - Kills the selected target or all monsters in radar range if "all" is specified.
 
-            // TODO: output
+            if (parameters?.Length > 0)
+            {
+                if (parameters[0] == "all")
```

Comment:
The more I think about it, the more I don't like having "/smite all" as a thing.  Not needed by players, not needed by admins (really), and creates some unnecessary design complications.